gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# 102 update email template

### Description

I updated the templates used to send confirmation emails to users. I also created `./setup` which is a setup script.

### Testing Directions

```
./setup
python3 manage.py runserver
```

OR

```
python3 manage.py makemigrations
python3 manage.py migrate
python3 manage.py set_name # Sets the domain name and display name.
python3 manage.py pull_seatgeek_data
python3 manage.py make_users
python3 manage.py runserver
```

1.  Navigate to `http://127.0.0.1:8000/accounts/signup/` and sign up.
1.  See that you get an email from ShowUp.
1.  Click the link in the email.
1.  Verify your email address.
1.  Log in.

### Checklist
